microsoft/ReportingServicesTools
# Fix #377 by loading creds from session unless provided

Fixes #377.

Changes proposed in this pull request:
 - If passing a WebSession and the Credentials are not supplied, the code checks if the credentials were originally set in New-RsRestSession call and retrieves those for use in the function

How to test this code:
 - I have tested rest code using web sessions with and without passing the credentials in the individual calls
 - I have also did some basic testing when using -DefaultCredentials parameter and it all seems to work as expected.
 
Has been tested on (remove any that don't apply):
 - Powershell 5 and above
 - SQL Server 2016 and above
